<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="com.linecorp.bot.client.ExceptionConverterTest" tests="3" skipped="0" failures="0" errors="0" timestamp="2017-11-02T20:18:48" hostname="Samson-PC" time="0.013">
  <properties/>
  <testcase name="convertTest" classname="com.linecorp.bot.client.ExceptionConverterTest" time="0.001"/>
  <testcase name="convertUnknownExceptionTest" classname="com.linecorp.bot.client.ExceptionConverterTest" time="0.001"/>
  <testcase name="exceptionInConvertFallbackedTest" classname="com.linecorp.bot.client.ExceptionConverterTest" time="0.011"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
